src/detect_breakends/junction_detection.cpp

```diff
@@ -5,6 +5,7 @@
 #include <seqan3/io/sequence_file/output.hpp>   // FASTA support
 
 #include "detect_breakends/bam_functions.hpp"                       // for hasFlag* functions
+#include "misc/statistics.hpp"                                      // for calculating average read depth
```

Comment:
Okay, this means, we would have
- /modules/METHOD_NAME/ for all methods
- /structures/ for our sturcures like junctions, breakends,....
- /variant_detections/
- /variant_parser/
- /misc/ for general functions?
We should discuss this in the todays meeting.